content/guides/developer-guides/delfin/driver-developer-guide/_index.md

```diff
@@ -0,0 +1,241 @@
+  ```
+
+* Extend base class StorageDriver defined in `<delfin path>/delfin/drivers/driver.py`, to implement a new driver.
+
+  ```python
+  # Copyright 2020 The SODA Authors.
+  #
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #     http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+
+
+  from delfin.drivers import driver
+
+
+  class SampleStorageDriver(driver.StorageDriver):
+    """SampleStorageDriver shows how to implement the StorageDriver,
+    it also plays a role as sample to fake data for being tested by clients.
+    """
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def reset_connection(self, context, **kwargs):
+        pass
+
+
+  ```
+
+* Implement all the interfaces defined in `<delfin path>/delfin/drivers/driver.py`, in the new driver.
+
+| Driver Interfaces | Input args | Output | Comment |
+| ------ | ------ | ------ | ------ |
+| **get_storage()** | `context`: Delfin framework context | Storage dict with schema defined in `delfin/db/sqlalchemy/model.py` `Storage` |  Get storage device information from storage system |
+| **reset_connection()** | `context`: Delfin framework context, kwargs: access info | None |  Reset connection with backend with new args |
+| **list_storage_pools()** | `context`: Delfin framework context | StoragePool dict with schema defined in `delfin/db/sqlalchemy/model.py` `StoragePool` |  List all storage pools from storage system |
+| **list_volume()** | `context`: Delfin framework context | Volume dict with schema defined in `delfin/db/sqlalchemy/model.py` `Volume` |  List all storage volumes from storage system |
+| **add_trap_config()** | `context`: Delfin framework context, `trap_config`: trap configuration | None |  Config the trap receiver in storage system |
+| **remove_trap_config()** | `context`: Delfin framework context, `trap_config`: trap configuration | None |  Remove trap receiver configuration from storage system |
+| **parse_alert()** | `context`: Delfin framework context, `alert`: alert to parse | Alert dict as shown in `delfin/drivers/driver.py` |  Parse alert data got from snmp trap server |
+| **list_alerts()** | `context`: Delfin framework context, `query_para`: optional, contains 'begin_time' and 'end_time' | Alert dict as shown in `delfin/drivers/driver.py` |  List all current alerts from storage system |
+| **clear_alert()** | `context`: Delfin framework context, `alert`: alert to clear | Success/Failure |   Clear alert from storage system |
+
+  ```python
+
```

Comment:
similar comment as above. We can highlight function name to be referred

Reply:
same as above